redash_make_default.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+# Trivial script to add Redash to the user login profile
+
+cat <<EOF >>~/.profile
+# Added by Redash 'redash_make_default.sh' script
+export COMPOSE_PROJECT_NAME=redash
+export COMPOSE_FILE=/opt/redash/docker-compose.yml
```

Comment:
If we want to carry over the `$REDASH_BASE_PATH` value from the setup script, we could probably change that `/opt/redash` text fragment into a token and have the setup script replace it.

I'll probably add that in tomorrow too, :smile:

Reply:
Done with the latest commit in this PR.